### DISCUSSION TITLE
Results pane not showing on startup

### DISCUSSION THREAD

**Discussion by rolandlkw** [dbeaver/dbeaver#11638] [Q&A] [answered]
I am currently running version 7.3.4.202101311042 on Windows 10.
In this version, when I open dBeaver, the results pane in the bottom part is opened as well and - of course - shows no data.

Ever since the last update - and now also on version 21 - when I open dBeaver, the results pane is hidden and only shows up, when I execute my first SQL.

Now I do understand that it is unneccessary to show the results pane on startup, as it contains no data, but it still somehow bothers me (I was just used to it over the years). Is there any way to restore the way it worked before, maybe through the settings (I didn't find anything), or do I have to just get used to it?

**Comment by ShadelessFox** [comment, marked as answer]
Now the results page is closed by default and there's no option to change it back, unfortunately.

But I hope you will get used to it :)
